aufgaben/funktionen/sternchen-funktion.py

```diff
@@ -0,0 +1,19 @@
+
+def erstezeile(zahl):
```

Comment:
Gutes, übersichtliches Programm! Top

Teilweise sind zusammengesetzte Wörter schwierig zu lesen. Lösung: `_`

```suggestion
def erste_zeile(zahl):
```

Reply:
ok vielen dank! kann ich einfach auf den commit knofp drücken auf githhub un es ändert sich automatisch? oder soll ich es manuell ändern? das ist nur eine frage um zu verstehen ob das bei github geht

Reply:
Das geht bei Github und wird im normalen Entwocklungsprozess genau so gemacht. Hier bin ich froh, wenn Sie die Änderungen gerade selber einpflegen, sonst muss ich jedesmal einen neuen Pullrequest öffnen 😉